src/utils/docs/category.yaml

```
@@ -0,0 +1,155 @@
+          example: 60d21b4667d0d8992e610c85
+        name:
+          type: string
+          description: Category name
+          minLength: 1
+          maxLength: 150
+          example: "Language"
+        appearance:
+          $ref: '#/components/schemas/CategoryAppearance'
+      required:
+        - name
+        - appearance
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)